sql/core/src/test/scala/org/apache/spark/sql/TPCDSQuerySuite.scala

```diff
@@ -353,8 +355,11 @@ class TPCDSQuerySuite extends QueryTest with SharedSQLContext with BeforeAndAfte
       classLoader = Thread.currentThread().getContextClassLoader)
     test(name) {
       withSQLConf(SQLConf.CROSS_JOINS_ENABLED.key -> "true") {
-        // Just check the plans can be properly generated
-        sql(queryString).queryExecution.executedPlan
+        // check the plans can be properly generated
+        val p = sql(queryString).queryExecution.executedPlan
+        // check the generated code can be properly compiled
+        val codes = debug.codegenCodeAndCommentSeq(p)
+        codes.map(c => CodeGenerator.compile(c))
```

Comment:
Is it helpful to show up the subtree which causes the compilation error?

Reply:
+1